source/support.tex

```diff
@@ -4238,9 +4238,10 @@
 \pnum
 The \tcode{partial_ordering} type is typically used
 as the result type of a three-way comparison operator\iref{expr.spaceship}
-that (a) admits all of the six two-way comparison operators (\ref{expr.rel}, \ref{expr.eq}),
-(b) does not imply substitutability,
-and (c) permits two values to be incomparable.
+that is intended to support
```

Comment:
add "for a type", consistent with the rest

Reply:
obsolete